src/plugins/fakeProfileThemes/index.tsx

```diff
@@ -232,10 +231,7 @@ export default definePlugin({
             if (settings.store.nitroFirst && user.themeColors) return user;
             const colors = decode(user.bio);
             if (colors) {
-                return virtualMerge(user, {
-                    premiumType: 2,
-                    themeColors: colors
-                });
+                return { ...user, premiumType: 2, themeColors: colors };
```

Comment:
```suggestion
                return Object.assign(user, { premiumType: 2, themeColors: colors });
```
you'd wanna return the same user object still, this is being passed to discord code so not many assumptions can be made

Reply:
this even still is probably not a great idea, i think `virtual-merge` is a fine solution